http/oidc/src/test/java/org/wildfly/security/http/oidc/OidcTest.java

```diff
@@ -102,13 +102,16 @@ public class OidcTest extends OidcBaseTest {
     // setting a high number for the token lifespan so we can test that a valid token from tenant1 can't be used for tenant2
     private static final int ACCESS_TOKEN_LIFESPAN = 120;
     private static final int SESSION_MAX_LIFESPAN = 120;
+    private static final int REFRESH_ACCESS_TOKEN_LIFESPAN = 10;
+    private static final int REFRESH_SESSION_MAX_LIFESPAN = 10;
```

Comment:
Just curious, is there a reason these low numbers are needed here if we are forcing the refresh to happen regardless?

Reply:
Git CI was failing on every test platform due to time over run for this test.  
With a lifespan of 120. testRefreshToken was taking 35sec to run and all the 
tests in the file 2+mins.  Reducing the lifespan for this test enabled the test file
to run in about 1.5 min.

Reply:
Do we know what was causing the test to take longer to run in that case? Because we're calling refreshToken with checkActive set to false, I'm just wondering why changing the access token lifespan would impact the test duration.

Reply:
I tried using a lifespan of 120 locally but I don't seem to see a significant difference in the overall time the test takes to run compared to when the lifespan is set to 10.

Reply:
I think we could try setting the lifespan back to 120 and see if the CI issue occurs. If it does, we can then take a closer look to see what's happening. I can't think of why the token lifespan would be affecting the test duration here.

Reply:
I've returned the test to it's original configuration.  All tests pass github CI

Reply:
Great. Are you able to re-trigger the test runs several times so we can confirm the tests pass consistently?

Reply:
Just an FYI, we've been re-triggering CI a bunch of times today and the CI runs have consistently passed.

Reply:
Well no they have not consistently passed.   There was 1 complete failure for any of the tests to run this afternoon.  I started a rerun and the tests passed.

Reply:
I haven't seen that. I've triggered runs at least 7 times so far and they have all completed without issues.

In the failure case you saw, it sounds like it wasn't specific to OIDC, just that none of the jobs were starting?

Reply:
> In the failure case you saw, it sounds like it wasn't specific to OIDC, just that none of the jobs were starting?

I found the job you mentioned. It was a "bind address already in use" intermittent failure for an unrelated test (the OIDC tests didn't get run).

Reply:
another job has failed.
https://github.com/wildfly-security/wildfly-elytron/actions/runs/14781730898
This job just failed with 1 Bind Address already in use: NET_Bind (window-11)
All other jogs canceled (I'm assuming because none responsive)
I have restarted the job.

Reply:
Thanks, Rebecca. Yes, I see the bind address in use message in the tests. Thanks for re-triggering. We should create an ELY issue for this intermittent bind address problem on Windows. Looks like the tests that I've seen fail with this intermitent issue are `ElytronAuthenticatorTest`, `MaskedPasswordSaslAuthenticationTest`, `JwtSecurityRealmTest`, `OAuth2SaslClientV10Test`, and `OAuth2SaslClientV11Test`.

Reply:
I don't see this as an elytron issue.  It's an issue with the github CI env.  We see this type of failure in the middle of the (U.S.) day not in the morning or evening.  Most likely the system is under heavy load at that time which causes communication delays.  Perhaps we should discuss this with the github CI maintenance team first.

Reply:
I'm wondering if it could be related to something happening in a previous test where something doesn't get stopped/cleaned up in time before these tests start or something like that.

Reply:
This is always a possibility.  Most likely it would be very hard to find as the failures are random and not reproducable locally.  Perhaps asking the performance team to evaluate elytron.